streamvc/dataset.py

```
@@ -0,0 +1,161 @@
+            target_audio: (segment_length,) target audio (different speaker)
+        """
+        # Load source audio
+        source_path = self.audio_files[idx]
+        source_audio = self.load_audio(source_path)
+        source_audio = self.random_crop(source_audio)
+        source_audio = self.augment_audio(source_audio)
+        
+        # Select random target audio (ensure different file)
+        target_idx = idx
+        while target_idx == idx and len(self.audio_files) > 1:
+            target_idx = random.randint(0, len(self.audio_files) - 1)
```

Ruff (0.15.2)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)